src/panels/lovelace/cards/hui-light-card.ts

```diff
@@ -93,36 +95,38 @@ export class HuiLightCard extends LitElement implements LovelaceCard {
           @click="${this._handleMoreInfo}"
         ></paper-icon-button>
 
-        <div id="light">
-          ${supportsFeature(stateObj, SUPPORT_BRIGHTNESS)
-            ? html`
-                <round-slider
-                  min="1"
-                  .value=${brightness}
-                  @value-changing=${this._dragEvent}
-                  @value-changed=${this._setBrightness}
-                ></round-slider>
-              `
-            : ""}
-          <ha-icon
-            class="light-icon"
-            data-state="${stateObj.state}"
-            .icon="${this._config.icon || stateIcon(stateObj)}"
-            style="${styleMap({
-              filter: this._computeBrightness(stateObj),
-              color: this._computeColor(stateObj),
-            })}"
-            @click="${this._handleClick}"
-          ></ha-icon>
+        <div id="controls">
+          <div id="slider">
+            ${supportsFeature(stateObj, SUPPORT_BRIGHTNESS)
+              ? html`
+                  <round-slider
+                    min="1"
+                    .value=${brightness}
+                    @value-changing=${this._dragEvent}
+                    @value-changed=${this._setBrightness}
+                  ></round-slider>
+                `
+              : ""}
+            <ha-icon
+              class="slider-center"
+              data-state="${stateObj.state}"
+              .icon="${this._config.icon || stateIcon(stateObj)}"
+              style="${styleMap({
+                filter: this._computeBrightness(stateObj),
+                color: this._computeColor(stateObj),
+              })}"
+              @action="${this._handleClick}"
+              .actionHandler=${actionHandler()}
+              tabindex="0"
+            ></ha-icon>
+          </div>
         </div>
 
-        <div id="tooltip">
+        <div id="info">
           <div class="brightness" @ha-click="${this._handleClick}">
-            ${brightness} %
-          </div>
-          <div class="name">
-            ${this._config.name || computeStateName(stateObj)}
+            &nbsp;&nbsp;${brightness} %
```

Comment:
Why are the  `&nbsp;&nbsp;` necessary?

Reply:
This has been removed. It's all done in `_dragEvent` anyway.